src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -152,6 +152,108 @@ UnderReview.prototype = {
   response: PropTypes.object || PropTypes.bool,
 };
 
+export function Approved(name) {
+  return (
+    <>
+      <div>
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 id="track-your-status-on-mobile" slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div>
+            <p className="vads-u-margin-y--0">
+              We reviewed your application and have determined that you’re
+              entitled to educational benefits under Survivors’ and Dependents’
+              Educational Assistance (DEA, Chapter 35). Your decision letter is
+              now available. A physical copy will also be mailed to your mailing
+              address.
+            </p>
+            <va-link
+              download
+              filetype="PDF"
+              href="https://www.va.gov"
+              text="Download your decision letter"
+            />
+          </div>
+        </va-alert>
+        <ConfirmationReceiptBox name={name} />
```

Comment:
Didn't catch this before, but the size of the "Application for VA education benefits (Form 22-5490)" heading is wrong. Its styling in the mockup match up with h3 styling. Either change the markup to an h3 or (if h4 is the appropriate heading category given the context) add the class `vads-u-font-size--h3`.

Reply:
Also, the date format of the received date is not correct. It should be "August 8, 2021", not "Aug 8, 2021"—the moth name should not be abbreviated. See the `formatReadableDate` in `vets-website/src/applications/fry-dea/helpers.jsx`.

Reply:
The default margin above the heading is too much combined with the padding of its container. I think adding the class `vads-u-margin-top--1` would make it closer to the mockup.

Reply:
resolved

Reply:
You successfully changed the heading to h3, so the `vads-u-font-size--h3` is redundant. You only need that class to add h3 styling to non-h3 elements for semantic reasons.

Reply:
The date format and top margin changes in the comments above still need implementing.

Reply:
the existing format from date-fns has been configured to output full month now

Reply:
top margin and padding are fixed now